backend/clubs/views.py

```diff
@@ -4083,7 +4090,7 @@ class UserViewSet(viewsets.ModelViewSet):
     queryset = get_user_model().objects.all().select_related("profile")
     permission_classes = [ProfilePermission | IsSuperuser]
     filter_backends = [filters.SearchFilter]
-    http_method_names = ["get"]
+    http_method_names = ["get", "post"]
```

Comment:
Just want to confirm we aren't currently using the fact that this was a GET only route as some kind of security feature or check and the serializer has proper checks on which fields are writable